### PR TITLE
Add Additional methods of creating staging directories

### PR DESCRIPTION
**Pull request by w0rk3r**

### CODE REVIEW

File: collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml [hunk 1]
```diff
@@ -24,6 +25,13 @@ platforms:
...
+        mkdir staged && echo %cd%\staged
     psh:
       command: |
-        New-Item -Path "." -Name "staged" -ItemType "directory" -Force | foreach {$_.FullName} | Select-Object
\ No newline at end of file
+        New-Item -Path "." -Name "staged" -ItemType "directory" -Force | foreach
+        {$_.FullName} | Select-Object
+      variants:
+        - command: |
+          [System.IO.Directory]::CreateDirectory("staged");
```
Comment (khyberspache) [inline collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml:37]: Spacing off

File: collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml [hunk 2]
```diff
@@ -24,6 +25,13 @@ platforms:
...
   windows:
+    cmd:
+      command: |
+        mkdir staged && echo %cd%\staged
     psh:
       command: |
-        New-Item -Path "." -Name "staged" -ItemType "directory" -Force | foreach {$_.FullName} | Select-Object
\ No newline at end of file
+        New-Item -Path "." -Name "staged" -ItemType "directory" -Force | foreach
+        {$_.FullName} | Select-Object
```
Comment (khyberspache) [inline collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml:34]: Make single line

**Review by khyberspache — CHANGES_REQUESTED** [review]

File: collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml (hunk 2 above)
Comment (w0rk3r) [reply collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml:34]: Done

File: collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml (hunk 1 above)
Comment (w0rk3r) [reply collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml]: Fixed

**Review by khyberspache — APPROVED** [review]

**Pull request merged**